apis/risingwave/v1alpha1/risingwave_types.go

```diff
@@ -148,6 +148,10 @@ type RisingWaveComponentGroupTemplate struct {
 	// Defaults to 30 seconds.
 	// +optional
 	TerminationGracePeriodSeconds *int64 `json:"terminationGracePeriodSeconds,omitempty"`
+
+	// metadata of the RisingWave's Pods.
+	// +optional
+	PodsMeta RisingWavePodTemplatePartialObjectMeta `json:"podsMetadata,omitempty"`
```

Comment:
I think it's fine to use `metadata` because the template itself is used to describe the Pod.